java/ql/lib/ext/java.io.model.yml

```diff
@@ -82,6 +82,7 @@ extensions:
       - ["java.io", "ByteArrayOutputStream", False, "writeTo", "", "", "Argument[this]", "Argument[0]", "taint", "manual"]
       - ["java.io", "CharArrayReader", False, "CharArrayReader", "", "", "Argument[0]", "Argument[this]", "taint", "manual"]
       - ["java.io", "CharArrayWriter", True, "toCharArray", "", "", "Argument[this]", "ReturnValue", "taint", "manual"]
+      - ["java.io", "CharArrayWriter", True, "toString", "()", "", "Argument[this]", "ReturnValue", "taint", "manual"]
```

Comment:
If a `df-generated` model is examined and determined to be correct then we make the provenance `df-manual`. Perhaps we could do something similar for speculative flow, to keep track of where the model came from? You'd also have to update [the validation code](https://github.com/github/codeql/blob/7de1182f92ea3122681aacae3d2e21fe93907915/shared/dataflow/codeql/dataflow/internal/FlowSummaryImpl.qll#L169), where I see the existing options. `sp` or `sf`?

Reply:
I'm not sure whether that's valuable - for now at least, all I got was a qualified name, which prompted me to have a look and add a manual model.